### PR TITLE
Dynamic scrollview height

### PR DESCRIPTION
**Pull request by lihter**
ReviewView and ScrollView now have dynamic height.

### CODE REVIEW

File: MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/MovieDetail/MovieDetailViewController+Design.swift [hunk 1]
```diff
@@ -64,7 +63,7 @@ extension MovieDetailViewController: DesignProtocol {
         contentView.snp.makeConstraints {
             $0.width.equalTo(UIScreen.main.bounds.width)
             $0.top.equalToSuperview()
-            $0.height.equalTo(1550)
+            $0.height.equalTo(0)
```
Comment (marinzg) [inline MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/MovieDetail/MovieDetailViewController+Design.swift]: This is what I ment with [this comment](https://github.com/lihter/Movie-App/pull/28#discussion_r686924288). But why do you need to set it to `0`?

File: MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/MovieDetail/MovieDetailViewController.swift [hunk 2]
```diff
@@ -36,6 +36,20 @@ class MovieDetailViewController: UIViewController {
...
+        contentView.subviews.forEach {
+            height += $0.frame.height
+        }
+        
+        height += 34 * offset + 30
+        
+        scrollView.contentSize = CGSize(width: UIScreen.main.bounds.width, height: height)
+        contentView.snp.updateConstraints {
+            $0.height.equalTo(height)
+        }
```
Comment (marinzg) [inline MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/MovieDetail/MovieDetailViewController.swift]: Scroll view can calculate this automatically if every subview has fixed height.

**Review by marinzg — CHANGES_REQUESTED** [review]

File: MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/MovieDetail/MovieDetailViewController+Design.swift (hunk 1 above)
Comment (lihter) [reply MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/MovieDetail/MovieDetailViewController+Design.swift]: [Here](https://github.com/lihter/Movie-App/pull/35/commits/bea08e4c4026e5a13e5d0634d3a88cd505dff0d6#diff-0e250b879344a24765e175866cb828b87d12e258a723eb41dcf4344c1dcd19fdL65-L66)

File: MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/MovieDetail/MovieDetailViewController.swift (hunk 2 above)
Comment (lihter) [reply MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/MovieDetail/MovieDetailViewController.swift]: [Here](https://github.com/lihter/Movie-App/pull/35/commits/bea08e4c4026e5a13e5d0634d3a88cd505dff0d6#diff-f5b01144bf2cdd8175add69b29bdfdcf3abf149a38fdd4929f07da30b107f914L37)

**Review by marinzg — APPROVED** [review]

**Pull request merged**